### ISSUE TITLE
i get low accracy on Kinetics

### ISSUE THREAD

**Issue by lee2h** [kenshohara/3D-ResNets-PyTorch#89]
from your paper, you get top-1 accuracy 60.1% on resnet-34, but i get 57.9%.
i use your pre-train model.
the result is video accuracy.
what's the problem?

i can't download some of kinetics dataset video. because some videos are blocked.
reason of low accracy is that? or any other reason?

**Comment by lee2h** [comment]
also, on UCF101, i get the accuracy 82%, 85% on pretrained ResNet18 and Resnet34. how get the your paper accuracy 84%, 87%?.

finetune on layer4 and fc (conv5_x and fc).

**Comment by victorkkkk** [comment]
How do you process the image? I think your image pre-process method may be different with kenshohara's. I face the same problem, and I get only 20% accuracy when I test kenshohara's UCF pretrain model on UCF dataset directly. (I wrote the pre-process by myself, only use cv2.cvtColor( ) ) @lee2h 
Could you please explain your image process method? @kenshohara

**Comment by lee2h** [comment]
@victorkkkk  maybe not.
i use kenshohara's resnet-18 (pretrained on Kinetics and he do training on UCF101 split 1). i get 84% accuracy, same as paper's result. 
but, i do training on UCF101 (pretrained on Kinetics from kenshohara), i get 82%.

i do not change any pre-processing code.

**Comment by cientgu** [comment]
> @victorkkkk maybe not.
> i use kenshohara's resnet-18 (pretrained on Kinetics and he do training on UCF101 split 1). i get 84% accuracy, same as paper's result.
> but, i do training on UCF101 (pretrained on Kinetics from kenshohara), i get 82%.
> 
> i do not change any pre-processing code.

I got the same issue, did you solve it? Many thanks.
